util/types/mydecimal.go

```diff
@@ -759,7 +771,7 @@ func (d *MyDecimal) Round(to *MyDecimal, frac int) (err error) {
 		pos := wordsFracTo*digitsPerWord - frac - 1
 		shiftedNumber := to.wordBuf[toIdx] / powers10[pos]
 		digAfterScale := shiftedNumber % 10
-		if digAfterScale > roundDigit || (roundDigit == 5 && digAfterScale == 5) {
```

Comment:
This operation `digAfterScale > int32(roundMode)` is unreasonable.
We should not use roundMode as a digit number.